src/core/reference/include/ngraph/runtime/reference/mvn.hpp

```diff
@@ -86,6 +86,25 @@ void mvn_6(const T* arg,
         divide(out, tmp_buffer.data(), out, in_shape, reduced_shape, op::AutoBroadcastType::NUMPY, true);
     }
 }
+
+template <typename T>
+AxisSet mvn_6_reduction_axes(const HostTensorPtr& axes_input, size_t rank) {
+    T* a = axes_input->get_data_ptr<T>();
+    auto v = std::vector<T>(a, a + axes_input->get_shape()[0]);
+    std::vector<size_t> axes(v.size(), 0);
+    for (size_t i = 0; i < v.size(); i++) {
+        if (v[i] < 0) {
+            if (rank + v[i] < 0) {
+                throw ngraph_error("Unexpected axis");
+            }
+            axes[i] = (size_t)(rank + v[i]);
+        } else {
+            axes[i] = (size_t)(v[i]);
+        }
+    }
+    return AxisSet(axes);
+}
+
```

Comment:
Why do we need this routine in .hpp if you use only from .cpp? I think we should move it to .cpp as well.

Reply:
Because it is template